### DISCUSSION TITLE
How do I make it cycle back to the first index?

### DISCUSSION THREAD

**Discussion by nyngwang** [L3MON4D3/LuaSnip#900] [Q&A] [answered]
Given this: (my `snip` is your `ls`)

```lua
snip.add_snippets('all', {
  snip.snippet("ternary", {
    node_insert(1, 'cond'), node_txt(' ? '), node_insert(2, 'then'), node_txt(' : '), node_insert(3, 'else')
  }),
})
```

Is it possible to jump from index `3` to `1` using `snip.jump(1)`?

**Comment by L3MON4D3** [comment, marked as answer]
No :/
Just by calling `snip.jump(-1)` multiple times

**Reply by nyngwang** [reply]
Can you support something like `snip.jump(-100)` and it would clamp the value so that I can jump to the beginning at once?

**Reply by L3MON4D3** [reply]
What are you trying to do?😅

**Reply by nyngwang** [reply]
Nevermind, I just realized that holding my keymap of `snip.jump(...)` can work. No problem now!

**Reply by L3MON4D3** [reply]
Alright, the manual solution👌👌
